src/explore-education-statistics-common/src/modules/find-statistics/components/charts/__tests__/MapBlock.test.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
+import React from 'react';
+
+import { render, wait } from 'react-testing-library';
+
+import domUtil from '@common-test/domUtil';
+
+import testData from '@common/modules/find-statistics/components/charts/__tests__/__data__/testBlockData';
+import MapBlock from '../MapBlock';
+import { DataBlockGeoJSON } from '@common/services/dataBlockService';
+
+describe('MapBlock', () => {
+  test('MapBlock renders', async () => {
+    expect(
+      testData.AbstractChartProps.meta.locations.E92000001.geoJson,
+    ).not.toBeUndefined();
```

Comment:
Not sure about the value of an assertion on test data like this. As it's just checking it's defined, it also doesn't really seem to provide much value.